### ISSUE TITLE
Adding more than 1 Wallbox to TL

### ISSUE THREAD

**Issue by Shrekmachine** [bassmaster187/TeslaLogger#677]
As 1 Wallbox is implemented, it would be good to consume 2 or more for bigger installations.
I have 2 TWC and would be glad to support (once I get TL running with Ubuntu 21.10.)

**Comment by Shrekmachine** [comment]
Hi @bassmaster187  As TL is running now again, I would be ready for a scenario test with 2 Wallboxes and 1 Tesla :)

**Comment by Adminius** [comment]
How can TeslaLogger know on which wallbox you are charging now?
Idea: check charging state from both wallboxes... 

but what happens if you have two cars charging on both wall boxes simultaneously? 
Which one should TL take as a data source for your car?

**Comment by Shrekmachine** [comment]
http://192.168.0.98/api/1/version throwing serial number

**Comment by Adminius** [comment]
yeah, it doesn't help... IP address is also unique for each wallbox...

How we can differentiate between two wallboxes?
Example: 
W1 with IP .98  and serial number 1111
W2 with IP .99 and serial number 2222

Tesla Car API says charging. we know where: at home.
but we don't know on which wallbox... 
We can ask W1: charging? -> no
W2 charging -> yes
Ok, we are charging on W2.

But if you charge 2 cars:
W1 charging -> yes
W2 charging -> yes

And now? Which car on which wallbox?

Does TWC provide charging time as a param?
We could check if charging time under 30-60 seconds, it should be the right one... 
But theoretically, if you start charging on both cars at the same time -> both charging timer are equal :/

**Comment by Shrekmachine** [comment]
{"contactor_closed":true,"vehicle_connected":true,"**session_s":69140,"** INC per second
false when not loading

**Comment by Adminius** [comment]
thx!
and how to solve a case with to cars started at the same time?

and cases with to different wallboxes...

**Comment by Shrekmachine** [comment]
2 cars at same time should be a rare case and be "accepted" issue... it would need to be in the same second.
And if 1 is already charging, this should be doable, I assume

**Comment by Adminius** [comment]
we have to find same timer for other supported wallboxes...
with shelly it is not possible
